src/domain/Timelines/ModularTimeline/style/wrappers.ts

```diff
@@ -66,7 +66,7 @@ const TimelineWrapper = styled.div`
 `
 
 const TimelineEventWrapper = styled.div<ITimelineEventWrapperProps>`
-  margin-left: 48px;
+  margin-left: 32px;
```

Comment:
could we use `Variables.Spacing.sXLarge`

Reply:
This is a magic number